pyairtable/api/abstract.py

```diff
@@ -146,21 +143,26 @@ def _all(self, base_id: str, table_name: str, **options) -> List[dict]:
             all_records.extend(records)
         return all_records
 
-    def _create(self, base_id: str, table_name: str, fields: dict, typecast=False, **options):
-
+    def _create(
```

Comment:
are we changing line length?

Reply:
No changes. I think this line was already over 88 chars, which is what we have as [our existing line limit today](https://github.com/gtalarico/pyairtable/blob/e367ae9880ff07e789ab8a462dc719ed9962cc2a/pyproject.toml#L7).

Reply:
Ah weird it thought we were linting on format and preventing merge but I guess that may have stopped working at some point